crates/services/gas_price_service/src/v1/da_source_service/dummy_costs.rs

```diff
@@ -34,4 +34,8 @@ impl DaBlockCostsSource for DummyDaBlockCosts {
             }
         }
     }
+
+    async fn set_last_value(&mut self, sequence_number: u32) -> DaBlockCostsResult<()> {
+        todo!()
```

Comment:
Seems like the intent here is not to add implementation but to show that this function should never be called in the test. If I got it right, maybe it'll be better to do `panic!("should not be called");`?

Reply:
I changed it to 
```rs
        unimplemented!("This is a dummy implementation");
```